app-frontend/src/app/pages/market/tool/tool.html

```diff
@@ -16,9 +16,13 @@ <h1 class="h3 page-title">
             <div uib-slide
                  index="$index"
                  ng-repeat="screenshot in $ctrl.testScreenshots track by screenshot.id">
-              <img ng-src="{{screenshot.url}}" >
+              <img ng-src="{{screenshot.url}}">
             </div>
           </div>
+          <div ng-if="$ctrl.tool && $ctrl.testScreenshots.length === 0"
+               class="image-placeholder" style="height: 480px; width: 800px;">
```

Comment:
We may want to create classes for the different sizes, since if we have similar layouts on multiple pages, it doesn't really make sense to have to do this every time.